common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -116,7 +113,7 @@ private static MojoPipeline loadMojoPipelineFromFile() {
         !Strings.isNullOrEmpty(MOJO_PIPELINE_PATH),
         "Path to mojo pipeline not specified, set the %s property.",
         MOJO_PIPELINE_PATH_PROPERTY);
-    log.info("Loading Mojo pipeline from path {}", MOJO_PIPELINE_PATH);
+    //    log.info("Loading Mojo pipeline from path {}", MOJO_PIPELINE_PATH);
```

Comment:
Why do we remove the logger? Is it because of Sage?

Reply:
oops was supposed to add the logger back... thanks for pointing this out!